### DISCUSSION TITLE
How to show Chinese data in ClickHouse connector

### DISCUSSION THREAD

**Discussion by samyScott** [trinodb/trino#13448] [Q&A] [answered]
when I use Trino search Chinese data in Clickhouse DB,  the result is unreadable codes
![mmexport1659404720231](https://user-images.githubusercontent.com/38185636/182273957-a88d2525-2c7c-472e-999b-e8c75e0ff7aa.png)

**Comment by ebyhr** [comment, marked as answer]
Did you try enabling `clickhouse.map-string-as-varchar`?

**Comment by samyScott** [comment]
yes

**Comment by ebyhr** [comment]
I can't reproduce the issue on upstream. Please provide steps to reproduce (`CREATE TABLE` and `INSERT` statements), ClickHouse and Trino version, `SHOW SESSION` result in Trino.

```
# ClickHouse
CREATE TABLE default.test (x String) ENGINE = MergeTree ORDER BY x;
INSERT INTO default.test VALUES ('汉语');

# Trino
SET SESSION clickhouse.map_string_as_varchar = true;
SELECT * FROM clickhouse.default.test;
  x
------
 汉语
```

**Reply by samyScott** [reply]
this method can solve the problem, maybe exist ways to solve the global problem, not just the current session

**Reply by ebyhr** [reply]
Please enable `clickhouse.map-string-as-varchar` in config property if you want to use it in global.

**Reply by samyScott** [reply]
okok，thanks
